code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -496,7 +496,7 @@
 	melee_damage_upper = 10
 	damage_transfer = 0.9
 	projectiletype = /obj/item/projectile/guardian
-	ranged_cooldown_cap = 1
+	ranged_cooldown_time = 1
```

Comment:
see if he changed the var name. It shouldn't of course if its the guardian i am thinking of it was a PAIN to balance the speed on that anyway. I think i actually put a click_delay in its code somewhere...